src/Stepper/StepButton.spec.js

```diff
@@ -48,49 +48,98 @@ describe('<StepButton />', () => {
+      const handleMouseEnter = spy();
+      const handleMouseLeave = spy();
+      const wrapper = themedShallow(
+        <StepButton
+          onMouseEnter={handleMouseEnter}
+          onMouseLeave={handleMouseLeave}
+        >
+          Step One
+        </StepButton>
+      );
 
-  it('sets the EnhancedButton backgroundColor on hover', () => {
-    const wrapper = themedShallow(
-      <StepButton>Step One</StepButton>
-    );
-    assert.strictEqual(wrapper.prop('style').backgroundColor, muiTheme.stepper.backgroundColor);
-    wrapper.setState({hovered: true});
-    assert.strictEqual(wrapper.prop('style').backgroundColor, muiTheme.stepper.hoverBackgroundColor);
+      it('should set the hovered state', () => {
+        wrapper.simulate('mouseEnter');
+        assert.strictEqual(wrapper.state('hovered'), true, 'should be hovered');
+        assert.ok(handleMouseEnter.calledOnce);
+        wrapper.simulate('mouseLeave');
+        assert.strictEqual(wrapper.state('hovered'), false, 'should not be hovered');
+        assert.ok(handleMouseEnter.calledOnce);
+        assert.ok(handleMouseLeave.calledOnce);
+      });
+
+      it('should set the EnhancedButton backgroundColor on hover', () => {
+        wrapper.setState({hovered: false});
+        assert.strictEqual(wrapper.prop('style').backgroundColor, muiTheme.stepper.backgroundColor);
+        wrapper.setState({hovered: true});
+        assert.strictEqual(wrapper.prop('style').backgroundColor, muiTheme.stepper.hoverBackgroundColor);
+      });
+    });
+
+    describe('handleTouchStart', () => {
+      const handleTouchStart = spy();
+      const handleMouseEnter = spy();
+      const wrapper = themedShallow(
+        <StepButton
+          onTouchStart={handleTouchStart}
+          onMouseEnter={handleMouseEnter}
+        >
+          Step One
+        </StepButton>
+      );
+
+      it('should set the touched state', () => {
+        assert.strictEqual(wrapper.state('touched'), false, 'should not be touched');
+        wrapper.simulate('touchStart');
+        assert.strictEqual(wrapper.state('touched'), true, 'should be touched');
+        assert.ok(handleTouchStart.calledOnce);
+      });
+
+      it('should not set the hovered state with touch set', () => {
+        wrapper.simulate('mouseEnter');
+        assert.strictEqual(wrapper.state('hovered'), false, 'should not be hovered');
+        assert.ok(handleMouseEnter.calledOnce);
+      });
+    });
+
+    it('should bubble callbacks used internally', () => {
+      const handleMouseEnter = spy();
+      const handleMouseLeave = spy();
+      const handleTouchStart = spy();
+      const wrapper = themedShallow(
+        <StepButton
+          onMouseEnter={handleMouseEnter}
+          onMouseLeave={handleMouseLeave}
+          onTouchStart={handleTouchStart}
+        >
+          Step One
+        </StepButton>
+      );
+      wrapper.simulate('mouseEnter');
+      assert.ok(handleMouseEnter.calledOnce);
```

Comment:
I would rather avoid using fancy stuff for the tests.
I like simplicity :grin:. What do you think about always using the same assertion?
```js
assert.strictEqual(handleMouseEnter.callCount, 1);
```